examples/jaynes_cummings.jl

```diff
@@ -0,0 +1,65 @@
+# We thank QuTiP developers for sharing the notebook on Jaynes Cummings model,
+# this is a direct translation of QuTiP version to solving in Julia using
+# QuDynamics.
+
+using QuBase
+using QuDynamics
+
+function jc_integrate(N, wc, wa, g, kappa, gamma, psi0, use_rwa, tlist)
```

Comment:
I guess this name comes from the qutip example, but here it doesn't make much sense. Maybe `jc_setup` would be better?